content/guides/local-hardening/_index.md

```diff
@@ -35,39 +53,43 @@ export OPENLANE2_TAG=2.0.8
 
 Note: the values of these values may change in the future - you can consult the [tt-gds-action](https://github.com/TinyTapeout/tt-gds-action/blob/main/action.yml) yaml for the latest values (look for the step named "Set up environment variables") 
 
-#### 2. Clone tt-support-tools
-
-Clone the [tt-support-tools](https://github.com/TinyTapeout/tt-support-tools) repo (`tt08` branch) inside the `tt` directory of your project:
+### 4. Install OpenLane 2
 
 ```sh
-cd ~/factory-test
-git clone -b tt08 https://github.com/TinyTapeout/tt-support-tools tt
+pip install openlane==$OPENLANE2_TAG
```

Comment:
I've updated the file omitting this specific recommendation, so far, but I think it would be useful to do this right after we go ahead with the modifications discussed here.

Reply:
Makes sense, thanks!